Update 02.kubernetes.md (#144)
neuvector/docs: versioned_docs/version-5.4/02.deploying/02.kubernetes/02.kubernetes.md

</li>
<li>

> Create the primary NeuVector services and pods using the preset version commands or modify the sample yaml below. The preset version invoke a LoadBalancer for the NeuVector Console. If using the sample yaml file below replace the image names and &lt;version> tags for the manager, controller and enforcer image references in the yaml file. Also make any other modifications required for your deployment environment (such as LoadBalancer/NodePort/Ingress for manager access etc). The below YAML needs to be changed for internal certificate changes if deployed from v5.4.2 or above. Refer this [YAML] (#kubernetes-deployment-yaml-for-v5.4.2-onwards)

Reviewer: ```suggestion
Create the primary NeuVector services and pods using the preset version commands or modify the sample yaml below. The preset version invoke a LoadBalancer for the NeuVector Console. If using the sample yaml file below replace the image names and &lt;version> tags for the manager, controller and enforcer image references in the yaml file. Also make any other modifications required for your deployment environment (such as LoadBalancer/NodePort/Ingress for manager access etc). The below YAML needs to be changed for internal certificate changes if deployed from v5.4.2 or above. Refer to this [YAML](#kubernetes-deployment-yaml-for-v542-onwards).
```

              - IPC_LOCK
```

### Kubernetes Deployment YAML for v5.4.2 onwards

> The following sample YAML is for version 5.4.2 onwards where we need to mount the internal certificates on Controller, Enforcer and scanner pods since we dont support hardcoded certificates anymore. Create internal-cert secret from the given link before deploying, https://open-docs.neuvector.com/deploying/production/internal

Reviewer: ```suggestion
The following sample YAML is for versions 5.4.2 and onwards where we need to mount the internal certificates on Controller, Enforcer and Scanner pods since we do not support hardcoded certificates anymore. Create the internal-certificate secret from the given link before deploying: https://open-docs.neuvector.com/deploying/production/internal.
```
